code/modules/food/drinkingglass/metaglass.dm

```diff
@@ -23,7 +23,7 @@
 			icon = R.glass_icon_file
 		else
 			icon = initial(icon)
-		
+
```

Comment:
Whitespace here should get re-added to prevent conflicts from editing a Polaris file.
```suggestion
		
```
This should make it easy to add it back in. IIRC, there's a way with GitHub that allows you to commit a suggested file line change, but forget the words involved with it.